### ISSUE TITLE
Layout.XML files missing from root of repository

### ISSUE THREAD

**Issue by ScooterTech** [LeifUK/OpenControls.Wpf#9]
The layout files that are referenced here: https://www.codeproject.com/Articles/5270871/A-WPF-Docking-Windows-Management-Library seem to be missing from the repository.

**Comment by cshawky** [comment]
Yep, I downloaded the project zip from Code Project and used them. Interesting observation is that the Code Project demos do not re dock properly, yet this current build here does. Nonetheless this is a very good set of examples: very useable.

**Comment by cshawky** [comment]
[Layouts.zip](https://github.com/LeifUK/OpenControls.Wpf/files/13925034/Layouts.zip)
Layouts from Code Project attached.
